Add more categories

Bre77 opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Since the addition of MyPlace support, this integration has more things it can do like Garage Doors, Lights, and general switches.


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Bre77 committed:
Add more categories

Bre77 committed:
Merge branch 'current' into aa-update

c0ffeeca7 commented on source/_integrations/advantage_air.markdown:
```diff
@@ -3,6 +3,9 @@ title: Advantage Air
   - Climate
+  - Cover
+  - Light
+  - Sensor
```
```suggestion
  - Sensor
  - Switch
  - Update
```

c0ffeeca7 commented on source/_integrations/advantage_air.markdown:
```diff
@@ -3,6 +3,9 @@ title: Advantage Air
 description: Instructions on how to integrate Advantage Air A/C controller into Home Assistant.
 ha_category:
   - Climate
```
```suggestion
  - Binary switch
  - Climate
```

c0ffeeca7 committed:
Add some more categories

as per core repo
 https://github.com/home-assistant/core/tree/b24fa4839db8a1e10d34d544ae6845838dc9c5f2/homeassistant/components/advantage_air

c0ffeeca7 reviewed (approved):
Thank you, @Bre77 :+1:
